src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,74 @@
+import styles from '@/components/Tabs/tabs.module.scss';
+import { useAppContext } from '@/context';
+import { isUserAuthorizedContext } from '@/context/isUserAuthorized';
+import { useTabs } from '@/hooks/useTabs';
+import { useContext } from 'react';
+import TaskList from '../tasks/TaskList/TaskList';
+import task from '@/interfaces/task.type';
+import fetch from '@/helperFunctions/fetch';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+import { IN_PROGRESS } from '@/components/constants/task-status';
+import { TASKS_URL } from '../constants/url';
+const { SUCCESS, ERROR } = ToastTypes;
+
+type TabsProps = {
+  filteredTasks: task[] | any
```

Comment:
Why `any` here?

Reply:
Hey, @shmbajaj you can check status-site channel on discord as to why this has been added.
surely not the right type inference but as @Maheima had a blocker and already was running late this was an immediate fix rather than a huge refactor!

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/417/#discussion_r1109292395
the conversation he is talking about is this https://discord.com/channels/673083527624916993/741616265050849282/1075645853596925962

Reply:
@Maheima add reason behind using `any` here.